bigtable/tests/unit/test_table.py

```diff
@@ -563,6 +563,30 @@ def test_read_row_complete(self):
         column.append(Cell.from_pb(chunk))
         self._read_row_helper(chunks, expected_result, app_profile_id)
 
+    def test_read_row_custom_retry(self):
+        from google.api_core import retry
+        from google.api_core.retry import if_exception_type
+        from google.cloud.bigtable.table import _BigtableRetryableError
+
+        credentials = _make_credentials()
+        client = self._make_client(
+            project="project-id", credentials=credentials, admin=True
```

Comment:
nit: please don't set `admin=True`.  Ideally, the client / connections should be closed, although I'm not sure we do that consistently.

Reply:
I removed admin=True from this test. All other tests in the file set admin=True when creating the client, should I create a separate PR and remove that from all the other tests? 
Regarding closing the client, none of the other tests do that

Reply:
It would be nice to have a separate PR for `admin=True` and closing the clients.